_includes/admin-users.md

```diff
@@ -0,0 +1,119 @@
+
+{% if include.product == "admin" %}
+  {% assign product_link="[Docker Admin](https://admin.docker.com)" %}
+  {% assign invite_button = "**Invite**" %}
+  {% assign export_button = "the **Action** icon and then select **Export users as CSV**" %}
+  {% if include.layer == "company" %}
+    {% assign member_navigation="Select your organization in the left navigation drop-down menu, and then select **Users**." %}
+    {% assign remove_button = "**Remove user**" %}
+  {% else %}
+    {% assign member_navigation="Select your organization in the left navigation drop-down menu, and then select *Members**." %}
+    {% assign remove_button = "**Remove member**" %}
+  {% endif %}
+{% else %}
+  {% assign product_link="[Docker Hub](https://hub.docker.com)" %}
+  {% assign member_navigation="Select **Organizations**, your organization, and then **Members**." %}
+  {% assign invite_button = "**Invite members**" %}
+  {% assign remove_button = "**Remove member**" %}
+  {% assign export_button = "**Export members**" %}
+{% endif %}
+
+## Invite members
+
+Owners can invite new members to an organization via Docker ID, email address, or via a CSV file containing email addresses. If an invitee does not have a Docker account, they must create an account and verify their email address before they can accept the invitation to join the organization. When inviting members, their pending invitation occupies a seat.
+
+### Invite members via Docker ID or email address
+
+Use the following steps to invite members to your organization via Docker ID or email address. To invite a large amount of members to your organization via CSV file, see the next section.
+
+1. Sign in to {{ product_link }}{: target="_blank" rel="noopener" class="_"}.
+2. {{ member_navigation }}
+3. Select {{ invite_button }}.
+4. Select **Emails or usernames**.
+5. Follow the on-screen instructions to invite members.
```

Comment:
**Thought**: I noticed we removed this instruction: `Separate multiple entries by a comma, semicolon, or space.` The UI doesn't tell you to do this, but it will tell you if an entry is invalid- still useful to include here maybe?